workspaces/bazaar/.changeset/witty-numbers-battle.md

```diff
@@ -0,0 +1,28 @@
+---
+'@backstage-community/plugin-bazaar-backend': minor
+---
+
+The plugin has been migrated to support the new backend system by default. If you were previously importing from `alpha`, you can now remove this.
+
+## Migrating
```

Comment:
I don't think we need all of this, we can just send Adopters of this plugin to the readme.